db/migrate/20171203052615_custom_files.rb

```diff
@@ -0,0 +1,13 @@
+class CustomFiles < ActiveRecord::Migration[5.1]
+  def up
+    create_table 'custom_files' do |t|
```

Comment:
__[Reek]__

[UncommunicativeVariableName] CustomFiles#up has the variable name 't'


<!-- Comment by [SideCI](https://sideci.com) -->